custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java

```diff
@@ -31,15 +31,25 @@
 public class JaegerThriftSpanExporterFactory implements ConfigurableSpanExporterProvider {
   private static final Logger log = LoggerFactory.getLogger(JaegerThriftSpanExporterFactory.class);
 
-  static final String SIGNALFX_AUTH_TOKEN = "signalfx.auth.token";
+  @Deprecated static final String SPLUNK_ACCESS_TOKEN_OLD = "signalfx.auth.token";
+  static final String SPLUNK_ACCESS_TOKEN = "splk.access.token";
```

Comment:
Why the heck did we agree on `splk` and not `splunk` ???

Reply:
I don't know... there's even a conversation on slack about using `SPLK_` and we both left 👍  there 😭 🤣   nobody knows why now 🤷‍♂️